<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelUrl">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>155</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>YouTube 주소를 입력하세요</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="youtubeUrlInput">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>27</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelFfmpeg">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>53</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FFmpeg 경로를 입력하세요</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ffmpegPathInput">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>71</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>150</y>
      <width>431</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputConsole">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>181</y>
      <width>431</width>
      <height>261</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Progress and logs will appear here...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DownloadPathInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>118</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDownload">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>다운로드 경로를 입력하세요</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
